app/pages/guide/template.njk

```diff
@@ -4,46 +4,49 @@
 {% from './components/subsection.njk' import subsection %}
 
 {% block body %}
-  <div class="nhsuk-grid-row nhsuk-width-container nhsuk-u-margin-bottom-8" data-test-id="guide-page-body">
-      <div class="nhsuk-u-padding-top-5" data-test-id="go-back-link">
-      {{ backLink({
-        "href": "/",
-        "text": "Go back to homepage"
-      }) }}
+  <div data-test-id="guide-page-body" class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full nhsuk-u-margin-bottom-3">
+      <div data-test-id="go-back-link">
+        {{ backLink({
+          "href": "/",
+          "text": "Go back to homepage"
+        }) }}
+      </div>
     </div>
-    <h1 class="nhsuk-u-margin-top-6 nhsuk-u-margin-bottom-4" data-test-id="guide-page-title">{{ title }}</h1>
-    <div class="nhsuk-grid-row">
-      <div class="nhsuk-grid-column-two-thirds">
-        <div class="nhsuk-u-font-size-24" data-test-id="guide-page-description">{{ description }}</div>
-        <div class="nhsuk-u-font-size-19" data-test-id="guide-page-subtext">
-          {{ subsection({
-            subSection: {
-              description: subtext
-            }
-          }) }}
+    <div class="nhsuk-grid-column-two-thirds">
+      <div class="nhsuk-u-margin-top-5 nhsuk-u-margin-bottom-0">
+        <h1 data-test-id="guide-page-title" class="nhsuk-u-margin-bottom-3">{{ title }}</h1>
+        <h2 data-test-id="guide-page-description" class="nhsuk-u-font-size-22">{{ description }}</h2>
+        <div data-test-id="guide-page-advice" class="nhsuk-inset-text nhsuk-u-padding-top-2 nhsuk-u-padding-right-4 nhsuk-u-padding-bottom-2 nhsuk-u-padding-left-4 nhsuk-u-margin-top-5 nhsuk-u-margin-bottom-6">
```

Comment:
yikes! that's a lot of classes. Also any reason why we are not using the nhsuk-frontend inset component?

Reply:
it's mainly the padding classes, the nhsuk doesn't seem to group their padding classes vertically and horizontally like other css frameworks do. And there is a loop going on inside of the inset text so it couldn't just be passed into the component

Reply:
if you put on `nhsuk-u-padding-2`then `nhsuk-u-padding-right-4 nhsuk-u-padding-left-4` that should work - only removes one class though